### DISCUSSION TITLE
Using systemd-units

### DISCUSSION THREAD

**Discussion by py-crash** [cupcakearmy/autorestic#95] [Q&A] [answered]
Hi! I wrote some systemd-units and timers for running and automating my backups: https://gitlab.com/py_crash/autorestic-systemd-units.

Would you be interested in a pull request adding something like this to the autorestic documentation?

**Comment by cupcakearmy** [comment, marked as answer]
Hey there, sry for the late reply and thanks for your work!
I'll add them to the [community part of the docs](https://autorestic.vercel.app/community) :)
❤️
